stories/Highlight.stories.js

```diff
@@ -0,0 +1,45 @@
+import React from 'react';
+import { storiesOf } from '@kadira/storybook';
+import { Highlight, Hits } from '../packages/react-instantsearch/dom';
+import { withKnobs, text } from '@kadira/storybook-addon-knobs';
+import { WrapWithHits } from './util';
+
+const stories = storiesOf('Highlight', module);
+
+stories.addDecorator(withKnobs);
+
+const Default = ({ hit }) => (
+  <p>
+    <Highlight attributeName="name" hit={hit} />
+  </p>
+);
+
+Default.propTypes = {
+  hit: React.PropTypes.object.isRequired,
+};
+
+const MarkHits = ({ hit }) => (
+  <p>
+    <Highlight
+      attributeName="name"
+      tagName={text('tag name', 'mark')}
+      hit={hit}
+    />
+  </p>
+);
+
+MarkHits.propTypes = {
+  hit: React.PropTypes.object.isRequired,
+};
+
+stories
+  .add('default', () => (
+    <WrapWithHits hasPlayground={false} linkedStoryGroup="Highlight">
+      <Hits hitComponent={Default} />
+    </WrapWithHits>
+  ))
+  .add('custom tag', () => (
```

Comment:
I noticed that the link to "play with props" doesn't work on this, but I'm not really sure why